src/NServiceBus.Core/Pipeline/PipelineCollection.cs

```diff
@@ -8,13 +8,22 @@ namespace NServiceBus
 
     class PipelineCollection
     {
-        public PipelineCollection(IEnumerable<TransportReceiver> pipelines)
+        public PipelineCollection()
```

Comment:
Interestingly it uses less allocations with the null check than with a dummy collection that needs to be passed in from outside.